ex4_js-objects-part2

ruminmaxim opened the pull request:
1. Object with prototype. Function shows properties of objects's prototype
2. Function makes object without prototype
3. Function trims spaces around the string
4. Function makes the first letter of string capital
5. Function searches string in another string and returns boolean result (contain / not contain)
6. Function makes the first letter of every word capital in a string
7. Function is shorting the string by set value
8. Function returns the string in lowerCamelCase notation
9. Function adds string to another string by set value
10. Function returns reversed string
11. Function counts symblos and amount of them in the string
12. Function rounds result of sum
13. Function returns random value from 0 to 100
14. Function returns random integer number in the set interval

ruminmaxim committed:
1. Object with prototype. Function shows properties of objects's prototype
2. Function makes object without prototype
3. Function trims spaces around the string
4. Function makes the first letter of string capital
5. Function searches string in another string and returns boolean result (contain / not contain)
6. Function makes the first letter of every word capital in a string
7. Function is shorting the string by set value
8. Function returns the string in lowerCamelCase notation
9. Function adds string to another string by set value
10. Function returns reversed string
11. Function counts symblos and amount of them in the string
12. Function rounds result of sum
13. Function returns random value from 0 to 100
14. Function returns random integer number in the set interval

aneelia commented on src/ex4_js-objects-part2/task-03.js:
```diff
@@ -0,0 +1,5 @@
+function spaceCut(str) {
+    return str.trim();
```
trim использовать не прикольно в учебке, переделай пожалуйста это задание

aneelia commented on src/ex4_js-objects-part2/task-10.js:
```diff
@@ -0,0 +1,7 @@
+function strReverse(str) {
+    var arr = str.split('');
```
можно в одну строку сделать)

aneelia commented on src/ex4_js-objects-part2/task-13.js:
```diff
@@ -0,0 +1,5 @@
+function random_0_100() {
+    return Math.random() * 100;
```
наверное, упущение в задании, но подразумевалось, что функция вернет целое число в этом диапазоне

ruminmaxim committed:
1. Function "Space Cut" made by another way
2. Function "strReverse" is shortened
3. Function "random_0_100" return integer
